students/ReemAlqaysi/Lesson04/mailroom.py

```diff
@@ -75,36 +72,35 @@ def exit_program():
     print("Bye!")
     sys.exit()  # exit the interactive script
 
-#send letterto all givers
+#send letter to all givers, file name is snake style
 def letter_to_all():
         for k,v in donor_list.items():
-            amount = v[len(v) - 1]
+            amount = k[len(v) - 1]
             fileName = k.replace(' ', '_').replace(',', '') + ".txt"
+            fileName = fileName.lower()
```

Comment:
my last comment about fileName needing to be snake_style is in regards to your variable naming.

fileName -> file_name


per pep8 (python style guide), all variable and function names follow lower case snake_style naming convention.

Reply:
currently the result I'm getting for example Reem Alqaysi >> reem_alqaysi is the file name .. not sure what else missing here.. can you advice please if I'm missing something?

Reply:
@Reem0330 per my previous comment we are not talking about name of your file.

We are talking about the variable name in your code. Pep8 is the style guide which advises code style to standardize code across developers. https://www.python.org/dev/peps/pep-0008/


your line of code should read:
` file_name = k.replace(' ', '_').replace(',', '') + ".txt"`

instead of 
`fileName = k.replace(' ', '_').replace(',', '') + ".txt"`

all variable names follow snake_case convention, for example:
`donorsList` vs `donors_list`